crates/extension/src/extension_manifest.rs

```diff
@@ -267,10 +267,12 @@ impl ExtensionManifest {
 
         let mut extension_manifest_path = extension_dir.join("extension.json");
         if fs.is_file(&extension_manifest_path).await {
-            let manifest_content = fs
-                .load(&extension_manifest_path)
-                .await
-                .with_context(|| format!("failed to load {extension_name} extension.json"))?;
+            let manifest_content = fs.load(&extension_manifest_path).await.with_context(|| {
+                format!(
+                    "failed to load {extension_name} extension.json, {:?}",
```

Comment:
```suggestion
                    "loading {extension_name} extension.json, {extension_manifest_path:?}"
```

Let's do something like this and throw away parasite `failed to` as we're in the error context already, of course we've failed.

Some other places below would benefit from the same.

Reply:
Good advice. Fixed.